package.json

```diff
@@ -2,16 +2,13 @@
   "name": "bpmn-js-tracking",
   "version": "0.1.0",
   "description": "",
-  "main": "dist/index.js",
-  "module": "dist/index.esm.js",
+  "main": "src/index.js",
   "files": [
-    "dist"
+    "src"
```

Comment:
Let's rename the `src` folder to `lib`, since we publish it as-is. This is also the pattern we use in [diagram-js](https://github.com/bpmn-io/diagram-js)